tests/src/python/test_qgssettingsentry.py

```diff
@@ -72,7 +72,7 @@ def test_settings_entry_base(self):
         self.assertEqual(settingsEntryVariant.valueAsVariant(), 43)
 
         # Settings type
-        self.assertEqual(settingsEntryVariant.settingsType(), QgsSettingsEntryBase.SettingsType.Variant)
+        self.assertEqual(settingsEntryVariant.settingsType(), Qgis.SettingsType.Variant)
```

Comment:
is this change just to remove deprecation warnings? Or to get the tests to pass?

Reply:
I used the sip patching technique technique and did a global replace. So there is no API break.